vars/vars.go

```diff
@@ -0,0 +1,43 @@
+package vars
+
+import (
+	"fmt"
+	"os"
+
+	"k8s.io/klog/v2"
+)
+
+type EnvVar struct {
+	env_var_name  string
+	default_value string
+	is_required   bool
+}
+
+func (ev EnvVar) GetValue() (string, error) {
+	os_env := ev.env_var_name
+	os_env_value, is_present := os.LookupEnv(os_env)
+
+	if is_present {
+		return os_env_value, nil
+	}
+
+	if ev.is_required {
+		return "", fmt.Errorf("%s: a required OS Environment is not present!", os_env)
+	}
+
+	// Return default endpoint
```

Comment:
Instead of endpoint, can we use some generic term like `param` so that this file can be used for all variables.

Reply:
Done!